### ISSUE TITLE
Integration with packrat

### ISSUE THREAD

**Issue by gacolitti** [road2stat/liftr#47]
Instead of manually entering in package version numbers, is it possible to use `packrat`? Maybe this can be accomplished with the `include` metadata?

**Comment by nanxstats** [comment]
Yes, I agree! That would be the direction to go. Had this documented in https://github.com/nanxstats/liftr/issues/3 by using `packrat:::fileDependencies` and `packrat:::fileDependencies.Rmd`.
